src/_pytest/pytester.py

```diff
@@ -1283,21 +1283,21 @@ def spawn(self, cmd: str, expect_timeout: float = 10.0) -> "pexpect.spawn":
 
 
 class LineComp:
-    def __init__(self):
+    def __init__(self) -> None:
         self.stringio = StringIO()
+        """:class:`python:io.StringIO()` instance used for input."""
 
-    def assert_contains_lines(self, lines2):
-        """Assert that lines2 are contained (linearly) in lines1.
-
-        Return a list of extralines found.
+    def assert_contains_lines(self, lines2: Sequence[str]) -> None:
+        """Assert that ``lines2`` are contained (linearly) in :attr:`stringio`'s value.
 
+        Lines are matched using :func:`LineMatcher.fnmatch_lines`.
         """
         __tracebackhide__ = True
         val = self.stringio.getvalue()
         self.stringio.truncate(0)
         self.stringio.seek(0)
         lines1 = val.split("\n")
-        return LineMatcher(lines1).fnmatch_lines(lines2)
+        LineMatcher(lines1).fnmatch_lines(lines2)
```

Comment:
why remove the return?

Reply:
hint: `LineMatcher.fnmatch_lines` is type-hinted (it returns None).

Reply:
> hint: ...

I don't think a "hint" is necessary, just explaining the reason is enough.

Reply:
Ok:

> `fnmatch_lines` returns nothing (see 4 clicks below)

Reply:
Thanks.